app/views/help/index.html.haml

```diff
@@ -0,0 +1,12 @@
+%ul
+  - @pages.each do |page|
+    - # this is a hack to capture the :title from each page: render the template, without actually outputting it
+    - render template: "help/#{page}"
```

Comment:
Have tried doing the rendering in a helper and the controller...can't figure out how to do it. Mind if we punt on it for now?